Assets/Scripts/TankControllerAndroid.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TankControllerAndroid : MonoSingletonGeneric<TankControllerAndroid>
```

Comment:
This class name should not be Android as Unity is a hybrid platoform where we will make the games for Android,WebGL aswell as Ios so any class name should not be like this  only TankController name is fine.